upgrade/helpers/tools.py

```diff
@@ -95,6 +96,45 @@ def host_pings(host, timeout=15):
             time.sleep(5)
 
 
+def host_ssh_check(host, timeout=15):
+    """This ensures the given IP/hostname pings succesfully.
+
+    :param host: A string. The IP or hostname of host.
+    :param int timeout: The polling timeout in minutes.
+
+    """
+    get_ip = subprocess.Popen(
```

Comment:
Call host_pings function from upgrade.tools here instead of repeating same things